cmd/client/main.go

```diff
@@ -1,10 +1,16 @@
 package main
 
 import (
-	"github.com/ASeegull/SmartFridge/examplePackage"
 	"fmt"
+	"log"
+	"net/http"
+
+	s "github.com/ASeegull/SmartFridge/staticServer"
 )
 
 func main() {
-	fmt.Println(examplePackage.Func("client"))
+	r := s.NewRouter()
+	port := ":" + s.GetPort()
```

Comment:
It would be better to have function like `GetAddr` that will return `[host]:[port]`